src/MCPServer/lib/archivematicaMCP.py

```diff
@@ -659,6 +660,13 @@ def _except_hook_log_everything(exc_type, exc_value, exc_traceback):
     t = threading.Thread(target=flushOutputs)
     t.daemon = True
     t.start()
+
+    # TODO: how many workers should we have? Should this be configurable?
```

Comment:
I'd like to get your thoughts here!

Reply:
I guess this comes down to how many package transfers might need to be started simultaneously? (at least for now with that being the only endpoint served by the workers). I don't know what is a good number to choose there but making it configurable seems sensible to me.